### PR TITLE
Add a cron job to garbage collect old test namespaces

### PR DESCRIPTION
**Pull request by huyhg**

### CODE REVIEW

**Comment by huyhg** [comment]
We apply this only once for the cluster.

File: testing/apptest-gc/manifests.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      | jq -r \
+        '.items[]
+         | { "name": .metadata.name, "time": (now - (.metadata.creationTimestamp|fromdate)) }
+         | select(.name | test("^apptest-\\w{8,16}$"))
+         | select(.time >= 1800)
+         | .name'))
+
+    echo "Number of eligible namespaces: ${#namespaces[@]}"
+
+    DRYRUN=${DRYRUN:-true}
```
Comment (trironkk) [inline testing/apptest-gc/manifests.yaml]: Looks like DRYRUN defaults to true - does this mean that the Cron is disabled by default?

Comment (huyhg) [reply testing/apptest-gc/manifests.yaml]: Oops, meant to set an environment variable for the container spec.

File: testing/apptest-gc/manifests.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  namespace: apptest-namespaces-gc
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gc-script
+  namespace: apptest-namespaces-gc
+data:
+  gc.sh: |-
+    set -eo pipefail
```
[the remark this answers is not in the document]
Comment (huyhg) [reply testing/apptest-gc/manifests.yaml]: Done.

**Pull request merged**